ads/google/a4a/traffic-experiments.js

```diff
@@ -62,19 +62,21 @@ const INTERNALLY_SELECTED_ID = '2088462';
  * @param {!ExperimentInfo} internalBranches experiment and control branch IDs to
  *   use when experiment is triggered internally (i.e., via client-side
  *   selection).
+ * @param {!string} controlWithDelayedId  Experiment ID string for the branch
```

Comment:
Can this not be included in the externalBranches and passed to maybeSetExperimentFromUrl via externalBranches.delayedImpressionControl (or something similar)?  Ideally ExperimentInfo would include within it the url mapping so that no special understanding of 1 = control, 2 = experiment, 3 = delayed control is necessary.  In fact, it could include whether it is a control or experiment and that would be used to determine if delayed or fast fetch should be used.